app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -239,4 +282,40 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
       sceneBucketJoinQuery.delete
     }
   }
+
+  /** Removes scenes from bucket
+    *
+    * @param sceneIds Seq[UUID] primary key of scenes to remove from bucket
+    * @param bucketId UUID primary key of bucket these scenes will be removed from
+    */
+  def deleteScenesFromBucket(sceneIds: Seq[UUID], bucketId: UUID)
+    (implicit database: DB): Future[Int] = {
+
+    val sceneBucketJoinQuery = for {
+      s <- ScenesToBuckets if s.sceneId.inSet(sceneIds) && s.bucketId === bucketId
+    } yield s
+
+    database.db.run {
+      sceneBucketJoinQuery.delete
+    }
+  }
+
+  /** Removes all scenes from bucket, then adds all specified scenes to it
+    *
+    * @param sceneIds Seq[UUID] primary key of scenes to add to bucket
+    * @param bucketId UUID primary key of bucket to remove all scenes from and add specified scenes to
+    * @return Scenes that were added
+    */
+  def replaceScenesInBucket(sceneIds: Seq[UUID], bucketId: UUID)
```

Comment:
In some functions the order of the parameters is `bucket, scene` and in others it is `scene, bucket`. Since Scene is always subservient to a Bucket in the Bucket datamodel, I propose we standardize to `bucket, scene` here.